.github/workflows/tracing.yml

```diff
@@ -13,7 +13,7 @@ concurrency:
 
 jobs:
   macos:
-    runs-on: macos-latest
+    runs-on: macos-latest-large
```

Comment:
This uses a more recent version that the version we currently guarantee to work on (which granted was already the case before, but that's also why things were failing)